modules/deploy/pages/deployment-option/self-hosted/kubernetes/cloud-instance-local-storage.adoc

```diff
@@ -0,0 +1,103 @@
+= Recommended Cloud Instance Types for Local NVMe SSD
+:description: Find the recommended cloud instance types that support local NVMe disks on Kubernetes managed services: Amazon EKS, Azure EKS, and Google GKE.
+
+{description}
+
+Redpanda Data recommends using locally attached NVMe devices with an XFS filesystem for the Redpanda data directory. NVMe devices offer high IOPS and minimal latency, while XFS offers enhanced performance with Redpanda workloads. For more details about cluster requirements and recommendations, see xref:./kubernetes-cluster-requirements.adoc[]
+
+== Amazon EKS
+
+*Default filesystem*: EKS defaults to the ext4 filesystem. Use XFS instead where possible.
+
+*Recommended instance types*:
+
+- *General purpose*: General-purpose instances provide a balance of compute, memory and networking resources, and can be used for a variety of diverse workloads. These instances are ideal for applications that use these resources in equal proportions such as web servers and code repositories.
```

Comment:
- *General purpose*: General-purpose instances provide a balance of compute, memory, and networking resources, and can be used for a variety of diverse workloads. These instances are ideal for applications that use these resources in equal proportions such as web servers and code repositories.

Reply:
I was unable to view/verify the output of anything in this section. While approving, please reach out to @micheleRP tomorrow with the preview link to have her review. Thx.

Reply:
This page no longer exists in this PR. It was removed and added in https://github.com/redpanda-data/docs/pull/36